src/AppInstallerCommonCore/FileLogger.cpp

```diff
@@ -24,25 +24,15 @@ namespace AppInstaller::Logging
     {
         m_name = GetNameForPath(filePath);
         m_filePath = filePath;
-
-        // Prevent inheritance to ensure log file handle is not opened by other processes.
-        FILE* filePtr;
-        if (!fopen_s(&filePtr, m_filePath.string().c_str(), "w") && filePtr != NULL)
-        {
-            SetHandleInformation((HANDLE)_get_osfhandle(_fileno(filePtr)), HANDLE_FLAG_INHERIT, 0);
-            std::ofstream outFile(filePtr);
-        }
-
-        m_stream.open(m_filePath, s_fileLoggerDefaultOpenMode);
+        OpenFileLoggerStream();
     }
 
     FileLogger::FileLogger(const std::string_view fileNamePrefix)
     {
         m_name = "file";
         m_filePath = Runtime::GetPathTo(Runtime::PathName::DefaultLogLocation);
         m_filePath /= fileNamePrefix.data() + ('-' + Utility::GetCurrentTimeForFilename() + s_fileLoggerDefaultFileExt.data());
-
-        m_stream.open(m_filePath, s_fileLoggerDefaultOpenMode);

```

Comment:
I can't comment on that line, but `s_fileLoggerDefaultOpenMode` is no longer used with these changes.